omod/src/main/java/org/openmrs/module/coreapps/page/controller/findpatient/FindPatientPageController.java

```diff
@@ -28,6 +31,7 @@ public void get(PageModel model, @RequestParam("app") AppDescriptor app, UiSessi
         model.addAttribute("heading", app.getConfig().get("heading").getTextValue());
         model.addAttribute("label", app.getConfig().get("label").getTextValue());
         model.addAttribute("showLastViewedPatients", app.getConfig().get("showLastViewedPatients").getBooleanValue());
+		model.addAttribute("showRegistrationLink", app.getConfig().get("showRegistrationLink").getTextValue());
```

Comment:
The 'schema' of an app config  is really the [template](https://github.com/openmrs/openmrs-module-coreapps/blob/master/omod/src/main/resources/apps/coreappsAppTemplates.json), there will be no wondering as long as the field is well documented in the template